apps/mailcatcher/templates/app/svc.yml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mailcatcher
+    service: app
+    version: "{{ mailcatcher_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "{{ mailcatcher_app_host }}"
```

Comment:
I would call it `mailcatcher` and remove this confusing variable (`mailcatcher_app_host` _versus_ `mailcatcher_host`).

Reply:
but there is already a "mailcatcher_host" here https://github.com/openfun/arnold/pull/120/files#diff-1d6d10331c2861156e52d71455fd392dR16 so I followed naming from the "redis" app.

Reply:
Sorry I did not meant to call your variable `mailcatcher`, but rather hard code the name:

```yaml
name: "mailcatcher"
```

Reply:
oh yeah you're right!